extensions/amp-facebook/0.1/test/validator-amp-facebook.out

```diff
@@ -68,5 +68,14 @@ PASS
 |        data-include-parent="true"
 |        data-href="https://www.facebook.com/zuck/posts/10102735452532991?comment_id=717538375088203">
 |    </amp-facebook>
+|    <!-- Example of a valid amp-facebook (embedded comment) -->
```

Comment:
This looks like a dupe of the above (i.e. bad merge by the tool).

Reply:
Reverted file.